cmp/ag-grid/AgGridModel.js

```diff
@@ -68,6 +68,16 @@ export class AgGridModel {
         });
     }
 
+    /**
+     * @returns {boolean} - true if the grid fully initialized and its state can be queried/mutated
+     */
+    // Intentionally not using @computed here because these api references may be updated if the grid
+    // is re-rendered, and @computed would not fire reactions since the return value would not have
+    // changed
+    get isReady() {
```

Comment:
I don't understand this logic?  Why does someone observing this getter care about the underlying api's?  If you are observing isReady() that's what you are observing.  If you need to observe the API directly, you should do that.

Reply:
Well I wanted any reactions on IsReady to be fired whenever the api references changed, because that is what would indicate that the grid state needs to be initialized if it is being managed by something else. It is possible that the grid would be re-initialized by being re-rendered, which means someone managing the state of this grid via this model would need to know about it and set the data/state again. I seem to remember having this same conversion with the initial implementation of this class where I had this same getter.. and we decided to remove it

Reply:
Yikes.  I also remember this.  Let's discuss.  Don't want to be going in circles here....